src/modules/workspaces/guards/workspaces-resources-in-items-in-behalf.guard.ts

```diff
@@ -88,7 +88,7 @@ export class WorkspacesResourcesItemsInBehalfGuard implements CanActivate {
 
     const actionHandler = this.getActionHandler(action);
 
-    const bypassActionCheck = request.isSharedItem;
+    const bypassActionCheck = request.query?.isSharedItem || false;
```

Comment:
This bypasses guard's security @jvedrson, please read this https://github.com/internxt/drive-server-wip/pull/407#issuecomment-2414125396 

Is sharedItem part in the request is used to let the workspaceGuard know that is it a sharedItem and bypass but we can not let the frontend decide that, that's something the backend should determine.